__tests__/main.test.ts

```diff
@@ -8,78 +8,76 @@
 
 import * as core from '@actions/core'
 import * as main from '../src/main'
+import * as tc from '@actions/tool-cache'
+import { chmod } from 'fs/promises'
 
-import * as gh from '@actions/github'
+jest.mock('fs/promises', () => ({
+  chmod: jest.fn().mockResolvedValue(undefined)
+}))
 
 const runMock = jest.spyOn(main, 'run')
 
-let errorMock: jest.SpiedFunction<typeof core.error>
 let getInputMock: jest.SpiedFunction<typeof core.getInput>
+let findCacheMock: jest.SpiedFunction<typeof tc.find>
+let downloadToolMock: jest.SpiedFunction<typeof tc.downloadTool>
+let cacheFileMock: jest.SpiedFunction<typeof tc.cacheFile>
+let addPathMock: jest.SpiedFunction<typeof core.addPath>
 let setFailedMock: jest.SpiedFunction<typeof core.setFailed>
-let setOutputMock: jest.SpiedFunction<typeof core.setOutput>
-
-// Mock the `getOctokit` method
-jest.mock('@actions/github', () => ({
-  getOctokit: jest.fn().mockReturnValue({
-    rest: {
-      repos: {
-        getReleaseByTag: jest.fn()
-      }
-    }
-  })
-}))
 
 describe('action', () => {
   beforeEach(() => {
     jest.clearAllMocks()
 
-    errorMock = jest.spyOn(core, 'error').mockImplementation()
-    getInputMock = jest.spyOn(core, 'getInput').mockImplementation()
+    getInputMock = jest.spyOn(core, 'getInput').mockImplementation(name => {
+      if (name === 'pkl-version') return '0.26.3'
+      return ''
+    })
```

Comment:
```suggestion
    getInputMock = jest.spyOn(core, 'getInput').mockImplementation(name => name === 'pkl-version' ? return '0.26.3' : return '')
```

Reply:
I was initially suspicious of using `''` as the falsy value for all of these, but it does seem like [that's what the actions toolkit has done](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L127-L128) as well 🤷